packages/common/src/ui/components/inputs/Autocomplete/Autocomplete.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Grid, Paper, Typography } from '@mui/material';
+import { Story } from '@storybook/react';
+import { styled } from '@mui/system';
+
+import { Autocomplete } from './Autocomplete';
+import { AutocompleteList } from './AutocompleteList';
+
+export default {
+  title: 'Inputs/Autocomplete',
+  component: Grid,
+};
+
+const StyledPaper = styled(Paper)({
+  height: 500,
+  padding: 10,
+  width: 400,
+});
+
+const options = Array.from({ length: 100 }).map((_, i) => ({
+  name: `${i}`,
+}));
+
+// TODO: Currently the styles are broken for this only within storybook
```

Comment:
this is super annoying but theres some weird styling issues which I can't seem to fix 😢  Only showing up in storybook